sw/device/tests/keymgr_sideload_otbn_test.c

```diff
@@ -100,20 +106,43 @@ static void test_otbn_with_sideloaded_key(dif_keymgr_t *keymgr,
 
   // Load the X25519 application.
   CHECK(otbn_load_app(otbn_ctx, kOtbnAppX25519) == kOtbnOk);
-
   // Run the OTBN app and retrieve the result.
   uint32_t result[8];
-  CHECK(run_x25519_app(otbn_ctx, result) == true);
+  CHECK(run_x25519_app(otbn_ctx, result, 0) == true);
```

Comment:
nit: using booleans instead of int, and also commenting the input parameters for readability:
```suggestion
  CHECK(run_x25519_app(otbn_ctx, result, /*expect_error=*/false) == true);
```

It might also be better to move the `CHECK` macro calls inside `run_and_check_x25519_app`. At the moment, there is a redudancy: we are both passing `expect_error` and also checking the function returns `!expect_error`.

Reply:
I change that function a bit now so it returns true so long as an error is expected, and false otherwise (will incorporate that it in once that other issue is fixed).